conjureup/ui/views/lxdsetup.py

```diff
@@ -106,20 +106,17 @@ def buttons(self):
 
     def build_info(self):
         items = [
-            Text("There was no LXD bridge found on your system "
-                 "which usually means this is your first time running "
-                 "LXD."),
+            Text("There was no LXD bridge found on your system or "
+                 "you are not part of the LXD user group which usually "
```

Comment:
I think this would be better if we were exact, and said "not part of the unix group 'lxd', which ..."